### PR TITLE
Migrate usages of @react-native-community/art to react-native-svg

### LINKED ISSUE #183: Doesn't work in Expo SDK36
Can't use it. Both iOS and Android crashes.
```
Invariant Violation: requireNativeComponent: "ARTShape" was not found in the UIManager.
```

### PR DESCRIPTION
**Pull request by frw**
@react-native-community/art could be deprecated soon: react-native-art/art#75
This pull requests replaces usages of @react-native-community/art with the well-supported react-native-svg library.

This should also resolve #183

### CODE REVIEW

**Comment by frw** [comment]
Tested with Example app:

https://user-images.githubusercontent.com/1888212/123537932-4f294580-d75c-11eb-8607-b3775f99943f.mp4

**Review by oblador — CHANGES_REQUESTED** [review]
Thanks a bunch for this, I know many have been looking forward to this PR. I've updated the Example app, fixed some minor lockfile misses so that it works with latest version of Xcode etc. 

Overall it looks good to me, one minor regression however is that the progress circle in your PR is different from master behavior. Should be an easy fix, see the expected animation in the video below:

https://user-images.githubusercontent.com/378279/123982426-500fe080-d9c3-11eb-99c1-74191c29c816.mp4

**Comment by frw** [comment]
@oblador Found the issue. It seems that passing in a rotation transform to the Svg component in Circle.js when it's no longer rotating causes it to be flipped. I'm not exactly sure why this is the case, but found a workaround to pass in an undefined style when there is no rotation and this seems to have fixed it.

https://user-images.githubusercontent.com/1888212/124013047-07304a00-da0c-11eb-9b95-169689502095.mp4

**Comment by asklar** [comment]
CC @marlenecota - any concerns about functionality in svg that this PR depends on, re: the Windows support you're adding, or should we be good?

File: README.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ Progress indicators and spinners for React Native using ReactART.
 
 `$ npm install react-native-progress --save`
 
-### ReactART based components
+### React Native SVG based components
```
Comment (mikehardy) [inline README.md:11]: There is still a reference to ReactART above :point_up:  on line 3 https://github.com/oblador/react-native-progress/pull/229/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3

More importantly: thank you for proposing this PR!

Comment (frw) [reply README.md:11]: Thanks - Fixed it!

**Review by mikehardy — APPROVED** [review]
Looks like it's in an "all comments addressed, needs hopefully-final review" state, so I approve, completely without any authority to do so :laughing:

**Comment by frw** [comment]
@oblador Could you take a quick look at this?

**Review by oblador — APPROVED** [review]
Champ!

**Comment by oblador** [comment]
Released in [5.0.0](https://github.com/oblador/react-native-progress/releases/tag/v5.0.0)

**Comment by mikehardy** [comment]
Likewise @oblador thanks for your work here and on react-native-vector-icons, I use both in my projects to great effect, I really appreciate it. Cheres

**Pull request merged**